appstudio-controller/controllers/appstudio.redhat.com/environment_controller_test.go

```diff
@@ -90,6 +90,11 @@ var _ = Describe("Environment controller tests", func() {
 							APIURL:                     "https://my-api-url",
 							ClusterCredentialsSecret:   secret.Name,
 							AllowInsecureSkipTLSVerify: allowInsecureSkipTLSVerifyParam,
+							ClusterResources:           true,
+							Namespaces: []string{
+								"namespace-1",
+								"namespace-2",
+							},
```

Comment:
It could be good to also have a test similar to what we had before, where we the environment that we created did not have these values set:
```go
							ClusterResources:           true,
							Namespaces: []string{
								"namespace-1",
								"namespace-2",
							},
```
(e.g. clusterresources is false, and namespace is empty, indicating that Argo CD has full access to the cluster.)

With the change it looks like (correct me if I'm wrong) we are now only testing the clusterResources = true and len(namespaces) > 0 case.

It should be possible to update the `createEnvironmentTest` function to support both (for example, a parameter that controls which case is being tested, and different expect values based on that parameter)

Reply:
I did add a test for this case at around line 681, but I'll update `createEnvironmentTest` anyway.